src/systems/isq/include/units/isq/mechanics.h

```diff
@@ -83,16 +82,18 @@ inline constexpr auto coefficient_of_static_friction = static_friction_coefficie
 QUANTITY_SPEC(kinetic_friction_factor, kinetic_friction_force / force, quantity_character::scalar);
 inline constexpr auto dynamic_friction_factor = kinetic_friction_factor;
 QUANTITY_SPEC(rolling_resistance_factor, force / force, quantity_character::scalar);
-// QUANTITY_SPEC(drag_coefficient, mag<2>* drag_force / (mass_density * pow<2>(speed) * area));
-// inline constexpr auto drag_factor = drag_coefficient;
+QUANTITY_SPEC(drag_coefficient, drag_force / (mass_density * pow<2>(speed) * area));  // TODO scalar or vector?
```

Comment:
This one is also a scalar. You can see that it's symbol is not in bold. It's not just `drag_force`, but it's magnitude. IIUC, the calculated characteristic is a vector, so it should be explicitly specified as a scalar, or somehow allow specifying that it's the magnitude of `drag_force`.

Reply:
> You can see that it's symbol is not in bold.

Oh, I did not notice that non-bold means scalar. I must admit I didn't have time to read the entire spec for now but I plan to do so when I am nearly ready to ensure that nothing is missing.